router/dealer.go

```diff
@@ -160,8 +160,11 @@ func NewDealer(strictURI, allowDisclose bool, metaClient wamp.Peer) Dealer {
 // Submit dispatches a Resister, Unregister, Call, Cancel, Yield, or Error
 // message to the dealer.
 func (d *dealer) Submit(sess *Session, msg wamp.Message) {
-	if msg == nil || sess == nil {
-		panic("nil session or message")
+	if msg == nil {
+		panic("dealer.Submit with nil message")
+	}
+	if sess == nil {
+		panic("dealer.Submit with nil session")
```

Comment:
panics like this are reserved for places that could ONLY be caused by programming errors.  If this were not an exported interface, I could remove it, but it means illegal misuse of the interface, therefore panic.